src/gc.c

```diff
@@ -135,6 +135,11 @@ typedef struct _bigval_t {
         size_t sz;
         uintptr_t age : 2;
     };
+#ifdef _P64
+    void *_padding[8 - 4];
+#else
+    void *_padding[16 - 4];
+#endif
```

Comment:
@tkelman Sorry, I missed your comment the first time around. The short answer is no, they're derived from pointer size + the number of other pointers in this struct. I've decided the best way to make this clear is to just [add comments](https://github.com/JuliaLang/julia/pull/15139/files#diff-31b93f4272eaeede3738531e90317e0cR138) that explain the padding.